### PR TITLE
Remove unneeded jump code

### PR DESCRIPTION
**Pull request by m0veax**
Regarding to the issue #683 unneeded code was removed to simplify the jump to mainstage

### CODE REVIEW

File: src/mainboard/sunxi/nezha/bt0/src/main.rs [hunk 1]
```diff
@@ -583,12 +583,8 @@ extern "C" fn main() -> usize {
...
 
     println!("Run payload at 0x{:x}", RAM_BASE);
-    unsafe {
-        let f: unsafe extern "C" fn() = transmute(RAM_BASE);
-        f();
-    }
-
-    // returns an address of dram payload; now cpu would jump to this address
+    
+	// returns an address of dram payload; now cpu would jump to this address
```
Comment (orangecms) [inline src/mainboard/sunxi/nezha/bt0/src/main.rs:587]: Here's the proposal: Instead of this, remove the other part. Then it's less functions. WDYT?

**Comment by orangecms** [comment]
see #683

**Comment by orangecms** [comment]
> so where is this one going?

We'll be discussing the whole thing and more work around the D1 / oreboot in general in person again in two weeks. I'm lucky to have found a few people interested in the project in our hackerspace here in Essen. :-)

**Comment by rminnich** [comment]
so ... this PR needs a more informative commit message. Image it's 2028, what will people be able to learn from this message? Very little, I think. Please if you can make the commit message more informative.

**Comment by m0veax** [comment]
Appreciate your feedback, we committed us to meet online tomorrow and get back on this topic ✌️

**Review by orangecms — APPROVED** [review]

**Pull request merged**